invenio_app_ils/cli.py

```diff
@@ -296,6 +296,35 @@ class DocumentGenerator(Generator):
         {"date": "2019-08-02", "place": "Geneva", "publisher": "CERN"},
         {"date": "2017-08-02", "place": "Hamburg", "publisher": "Springer"},
     ]
+    ISBNS = [
```

Comment:
Fantastic! Can you copy paste the same for the ISSN for the series to show a cover?